Task03/Task03/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task03
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int n, unit, ten, hundred;
+            Console.WriteLine("В трехзначном числе x зачеркнули вторую цифру. Когда к оставшемуся двузначному числу приписали эту цифру справа, получилось число n.");
+            Console.WriteLine("Введите число n(100<=n<=999, и число десятков не равно нулю)");
+            n = int.Parse(Console.ReadLine());
+            unit = n % 10;
+            hundred = n / 100;
+            ten = (n - hundred*100)/ 10;
```

Comment:
Переменные лучше декларировать рядом с их первым использованием (или сразу декларировать и инициализировать). Так код читается легче.
```suggestion
            Console.WriteLine("В трехзначном числе x зачеркнули вторую цифру. Когда к оставшемуся двузначному числу приписали эту цифру справа, получилось число n.");
            Console.WriteLine("Введите число n(100<=n<=999, и число десятков не равно нулю)");
            int n = int.Parse(Console.ReadLine());
            int unit = n % 10;
            int hundred = n / 100;
            int ten = (n - hundred*100)/ 10;
```